mods/default/trees.lua

```diff
@@ -560,9 +566,12 @@ function default.sapling_on_place(itemstack, placer, pointed_thing,
 			interval) then
 		minetest.record_protection_violation(pos, player_name)
 		-- Print extra information to explain
+--		minetest.chat_send_player(player_name,
+--			itemstack:get_definition().description .. " will intersect protection " ..
```

Comment:
What ist this pollution?  I am going nuts...

Reply:
Oh calm down old boy. This is just th eold code made comments for documentray purposes intentionally by myself.